packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -266,17 +304,22 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         return null;
     }
 
+    /* tslint:disable */
     private getTocItemCategory(name: string | any, icon?: JSX.Element): JSX.Element {
+        let renderLayout = icon ? <span className={this.props.managedClasses.site__paneToggleButtonIconLayout}><div className={this.props.managedClasses.site__paneToggleButtonIcon}>{icon}</div></span> : null;
```

Comment:
It seems odd to me that here and below we are wrapping a div inside a span. Is there a reason for this? This seems reversed.

Reply:
After reviewing the spec, the HTML5 spec doesn't allow for flow content (div) to be included within a span.